scripts/tools/GlobalTimestamps.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Tool Experimental Extension for Global Metrics Timestamps"
+version: "1.5"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "Global Metrics Timestamps Experimental Extension Name"
+version: "1.5"
+name: $T_GLOBAL_METRICS_TIMESTAMPS_EXP_NAME
+value: '"$XT_experimental_global_metrics_timestamps"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Global Metrics Timestamps Experimental Extension Version(s)"
+version: "1.5"
+name: $x_global_metrics_timestamps_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Returns global device timestamps synchronized with metrics timesstamps, optionally mapped to host."
+class: $tMetricGroup
+name: GetTimestampsExp
+decl: static
+details:
+    - "The application may call this function from simultaneous threads."
+params:
```

Comment:
for consistency with oher APIs the order of params should as 

`ze_result_t zetDeviceGetMetricTimestamps(ze_device_handle_t hDevice, uint64_t* globalTimestamp, uint32_t *pCount,  metric_group_handle_t *phMetricGroups, uint64_t* metricTimestamps);`

Reply:
@jandres742 would you please share how the signature of the API will look like with the latest changes? I'm not seeing the device hangle as input or the metric group timestamp .

Reply:
Based on my "assumption" of how this .yml will be processed, it might look like below

```
zetGetGlobalTimestampsExp(
                    zet_metric_group_handle_t hMetricGroup, 
                    ze_bool_t synchronizedWithHost, 
                    uint64_t* globalTimestamp, 
                    uint64_t* metricTimestamp)
```
I assume the return type would be **ze_result_t**

@jandres742 could confirm

Reply:
why are we adding _metric_global_timestamps_resolution_exp_t ?

Reply:
>why are we adding _metric_global_timestamps_resolution_exp_t ?

We need to know the timer frequency and when the timer will overflow (valid bits).
These will be used for mapping the timestamp to other timers (host clock)

Reply:
perfect!